build/build.go

```diff
@@ -500,7 +500,9 @@ func BuildWithResultHandler(ctx context.Context, nodes []builder.Node, opts map[
 						resultHandle, rr, err = NewResultHandle(ctx, cc, *so, "buildx", buildFunc, ch)
 						resultHandleFunc(dp.driverIndex, resultHandle)
 					} else {
+						span, ctx := tracing.StartSpan(ctx, "Build")
```

Comment:
Does casing matters?

```suggestion
						span, ctx := tracing.StartSpan(ctx, "build")
```

Reply:
🤷 Is there a convention? Or should we define one?

Reply:
Don't think there is any convention but it's lowercasing from what I see on buildkit repo: https://github.com/search?q=repo%3Amoby%2Fbuildkit%20tracing.StartSpan&type=code